lib/ddtrace/notifications.rb

```diff
@@ -0,0 +1,34 @@
+module Datadog
+  class Notifications
```

Comment:
Given the purpose of this module is to aid integrations into frameworks/libraries that implement `ActiveSupport::Notifications`, would it make sense to move this into that `Contrib` namespace, where that integration support lives? Something like `Datadog::Contrib::ActiveSupport::Notifications`?

I think one of our goals is to reduce framework dependencies in the core library, isolating them to the `Contrib` namespace, where we can optionally load code based on the presence of libraries (e.g. ActiveSupport.) In such a case, we could then modify the `Racecar::Patcher` to `require 'ddtrace/contrib/active_support/notifications'` when it verifies the presence of ActiveSupport and patches Racecar.

Reply:
Good idea.